batavia/modules/time.js

```diff
@@ -239,21 +248,28 @@ time.localtime = function(seconds) {
         throw new exceptions.TypeError.$pyclass('localtime() takes at most 1 argument (' + arguments.length + ' given)')
     }
     var date
-    if (arguments.length === 1) {
+    if (arguments.length === 1)
+    {
         // catching bad types
-        if (types.isinstance(seconds, [types.Complex])) {
+        if (types.isinstance(seconds, [types.Complex])) 
+        {
             throw new exceptions.TypeError.$pyclass("can't convert " + type_name(seconds) + ' to int')
```

Comment:
At column 5: [(brace-style) Opening curly brace does not appear on the same line as controlling statement.](http://eslint.org/docs/rules/brace-style)